NewForce-at-Mountwest/nutshell-team-stumbletoe
# Finished Chat. Added edit functionality.

Added the ability to edit messages created by the user.

To Test:
1) make sure there are several messages in the database from several different users.

2) Log in to an account

3) Scroll to the chat section and type in a message or scroll to a message made by the user.

4) you should see a button that says "edit" next to the message (and only on messages typed by the user)

5) click the edit button

6) The area where you type a new message should be blank and the button next to it should now say "Save Edit".

7) Type something into the input field.

8) Click the "save edit" button

9) The field should be back to the original placeholder and the button should once again say "New Message". 

10) The message you clicked the 'edit' button on should have the new text you typed.
